replica/backup.go

```diff
@@ -77,7 +93,8 @@ func (rb *Backup) OpenSnapshot(snapID, volumeID string) error {
 
 func (rb *Backup) assertOpen(id, volumeID string) error {
 	if rb.volumeID != volumeID || rb.snapshotID != id {
-		return fmt.Errorf("Invalid state volume [%s] and snapshot [%s] are open, not [%s], [%s]", rb.volumeID, rb.snapshotID, id, volumeID)
+		return fmt.Errorf("Invalid state volume [%s] and snapshot [%s] are open, not [%s], [%s]",
```

Comment:
Don't need this. #363 already have a similar fix. Also, the order of snapshot ID and volume ID is wrong.

Reply:
I didn't understand. Are you saying we don't need this function?
I just inserted a new line for arguments

Reply:
I meant #358 . Yeah, we don't need this since #358 already has a similar fix.